src/models/request.ts

```diff
@@ -32,6 +32,10 @@ export interface Request {
     pruneRefinements: boolean;
     disableAutocorrection: boolean;
     wildcardSearchEnabled: boolean;
+
+    // Tracking
```

Comment:
The comments above use a leading lowercase letter. I think we should do the same here.